src/main/java/betterquesting/client/gui2/GuiHome.java

```diff
@@ -65,36 +66,42 @@ public GuiHome(GuiScreen parent)
 	}
 	
 	@Override
-	public void initPanel()
-	{
+	public void initPanel() {
 		super.initPanel();
-		
+
+		// If we come to home gui - we set skip home to false
+		if (BQ_Settings.skipHome) {
+			ConfigHandler.config.get(Configuration.CATEGORY_GENERAL, "Skip Home", false).set(false);
+			ConfigHandler.config.save();
+			BQ_Settings.skipHome = false;
+		}
```

Comment:
This seems to no longer matter, as going to the home page opens up at the quest lines again even after setting the config option. Based on discussion on discord, setting the config option should probably just be removed.